modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SwiftCodegen.java

```diff
@@ -115,10 +117,10 @@ public SwiftCodegen() {
         typeMapping.put("string", "String");
         typeMapping.put("char", "Character");
         typeMapping.put("short", "Int");
```

Comment:
Second thought, forgot about this problem:
If a server mistakenly sends back 64bit integer in spite of `short` in the api spec, 64bit platform clients can handle with `Int`, but 32bit platform clients can't. This kind of mistake can go by unnoticed.
So maybe `short` should be mapped to `Int16` ?

I also couldn't find `short` in the swagger spec. Maybe this line should be deleted if `short` really does not (and will not) exist in the specs.
Does anyone know where this `short` comes from?